src/sentry/web/api.py

```diff
@@ -332,19 +348,29 @@ def process(self, request, project, key, auth, helper, data, **kwargs):
             sender=type(self),
         )
 
-        should_filter = helper.should_filter(project, data, ip_address=remote_addr)
+        should_filter = helper.should_filter(
+            project, data, ip_address=remote_addr)
         if should_filter[0]:
             tsdb.incr_multi(
                 [
                     (tsdb.models.project_total_received, project.id),
                     (tsdb.models.project_total_blacklisted, project.id),
-                    (tsdb.models.organization_total_received, project.organization_id),
-                    (tsdb.models.organization_total_blacklisted, project.organization_id),
+                    (tsdb.models.organization_total_received,
+                     project.organization_id),
+                    (tsdb.models.organization_total_blacklisted,
+                     project.organization_id),
                     (tsdb.models.key_total_received, key.id),
                     (tsdb.models.key_total_blacklisted, key.id),
                 ]
             )
-            metrics.incr('events.blacklisted', tags={'reason': should_filter[1]})
+            try:
+                tsdb.incr(
+                    FILTER_STAT_KEYS_TO_VALUES[should_filter[1]], project.id)
```

Comment:
This should probably be included in the list starting on L355 to avoid an unnecessary round trip.

Reply:
Since `should_filter[1]` is used multiple times, in this block, it's probably worth assigning to a local variable (e.g. `filter_reason`) just for clarity's sake.

Reply:
👍